modules/core/src/main/java/org/apache/ignite/internal/processors/metric/GridMetricManager.java

```diff
@@ -840,9 +841,10 @@ private double getCpuLoad() {
             long cpuTime;
 
             try {
-                cpuTime = U.<Long>property(os, "processCpuTime");
+                com.sun.management.OperatingSystemMXBean sunOs = (com.sun.management.OperatingSystemMXBean) os;
+                cpuTime = sunOs.getProcessCpuTime();
             }
-            catch (IgniteException ignored) {
+            catch (ClassCastException ignored) {
```

Comment:
What if `sunOs.getProcessCpuTime` throws RE?

Reply:
I don't think it ever happens, but I don't mind replacing `ClassCastException` with `Exception`.